src/ucp/proto/proto_common.inl

```diff
@@ -362,6 +362,14 @@ ucp_proto_request_pack_rkey(ucp_request_t *req, ucp_md_map_t md_map,
                   md_map);
     }
 
+    if ((memh->flags & UCP_MEMH_FLAG_HAS_GVA) &&
+        (ucp_ep_config(req->send.ep)->key.err_mode ==
+         UCP_ERR_HANDLING_MODE_PEER)) {
+        if (ucp_memh_disable_gva(memh, md_map) != UCS_OK) {
+            return 0;
```

Comment:
how does proto-infrastructure handle this? I think need to abort request here.

Reply:
it will send RTS/RTR without rkey, just with address

Reply:
it seems weird to me, I would expect that we need handle error from `ucp_memh_disable_gva`

Reply:
AFAIU like in case of error in ucp_rkey_pack_memh() we fallback to protocol without rkey

Reply:
disable_gva should return void, if it fails memh->md_map will be partial